sdk/python/kfp_tekton/tekton.py

```diff
@@ -341,11 +341,15 @@ def __init__(self, task_type: str = RUN_TYPE,
                 api_version: str = DEFAULT_APIVERSION,
                 is_finally: bool = False,
                 parallelism: int = None,
-                params: Dict[str, Union[dsl.PipelineParam, str, int]] = {}):
+                params: Dict[str, Union[dsl.PipelineParam, str, int]] = {},
+                annotation: Dict[str, str] = {},
```

Comment:
It seems like the plural (`annotations`) would fit better here?

Reply:
nice catch and updated!